### PR TITLE
fix: RPCInvocation.ServiceKey use PATH_KEY instead of INTERFACE_KEY

### LINKED ISSUE #1077: using different attachmentKey with java server when buiding ServiceKey
**What happened**:

func RPCInvocation.ServiceKey() using **interface** to generate the exporter key, while java use **path**

**What you expected to happen**:

dubbo-go use **path** too

**How to reproduce it (as minimally and precisely as possible)**:

consumer pass different value of interface and path

**Anything else we need to know?**:

image
image

### PR DESCRIPTION
**Pull request by tweitjss**
**What this PR does**:

fix RPCInvocation.ServiceKey use INTERFACE_KEY which is different from java

**Which issue(s) this PR fixes**:

Fixes #1077 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by zouyx — COMMENTED** [review]
can you help to add a test case for this change?

**Comment by zouyx** [comment]
> **What this PR does**:
> 
> fix RPCInvocation.ServiceKey use INTERFACE_KEY which is different from java
> 
> **Which issue(s) this PR fixes**:
> 
> Fixes #1077
> 
> **Special notes for your reviewer**:
> 
> **Does this PR introduce a user-facing change?**:
> 
> ```
> NONE
> ```

@tweitjss  Thanks for you contribution.

**Comment by AlexStocks** [comment]
firstly, pls move ur target branch from master to 3.0. sencondly, pls add an ut.

File: protocol/invocation/rpcinvocation_test.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package invocation
```
Comment (AlexStocks) [inline protocol/invocation/rpcinvocation_test.go:1]: pls add apache license on top of this file.

**Review by AlexStocks — APPROVED** [review]

**Review by georgehao — APPROVED** [review]

**Review by watermelo — APPROVED** [review]

**Pull request merged**